### DISCUSSION TITLE
guide/1-DDNS-GO

### DISCUSSION THREAD

**Discussion by giscus** [yyyytawa/yyyytawa.github.io#3] [Announcements]
# guide/1-DDNS-GO

简介 DDNS-GO 是一款使用GO语言，开源DDNS软件，意在解决家宽公网IP不固定，每回访问都要换IP。用了之后只需要弄一个固定的域名即可 在开始本篇教程前，你需要 托管在DDNS-GO支持的DNS服务商的域名(如Cloudflare，阿里云，DNSpod(腾讯云)等等)(当然你也需要有一个域名才行) 现在已经申请不了了...... 你要有公网IP...

https://docs.yyyyt.eu.org/guide/1-DDNS-GO.html

**Comment by yyyytawa** [comment]
aaaa

**Comment by yyyytawa** [comment]
展示图片: ![img](https://img.yyyyt.eu.org/test/test1.jpg)

**Reply by yyyytawa** [reply]
哈哈哈哈
